tts/global.def.lua

```diff
@@ -42,3 +42,34 @@ function printToAll(message, color) end
 ---@return string
 function stringColorToRGB(player_color) end
 
+
+---@shape tts__SnapPoint
+---@field position tts__Vector Position of the snap point. The position is relative to the entity's center (a local position).
+---@field rotation tts__Vector Rotation of the snap point. The rotation is relative to the entity's rotation (a local rotation).
+---@field rotation_snap boolean If the snap point is a "rotation" snap point.
+
+---@overload fun(parameters: table):string
+---@param parameters tts__SnapPoint[]
+---@return string
+function setSnapPoints(parameters) end
+
+--- Returns a table of sub-tables, each sub-table representing one snap point.
```

Comment:
As above, preference would be to use the term array, where it makes sense to do so.